### PR TITLE
Create linit 2018

### PR DESCRIPTION
**Pull request by 96sudeshnasen**

### CODE REVIEW

File: _linit/linit 2018 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: '2018'
+link: 'https://drive.google.com/open?id=11baSJJTczvTSkTB_8Z2FH0rR3z_BizpP'
```
Comment (Compro-Prasad) [inline _linit/linit 2018]: This link shows 404.

**Comment by Compro-Prasad** [comment]
Just a question. What did you use to make this PDF?

**Comment by 96sudeshnasen** [comment]
Sneha Di did the final compilation.

**Comment by Compro-Prasad** [comment]
Change 2018 linit image and commit. You have added 2017's screenshot I suppose.

**Comment by 96sudeshnasen** [comment]
'Uploads are disabled' 
I'm not being able to upload the new cover picture to add it to the commit

**Comment by Compro-Prasad** [comment]
Didn't add the screenshot?

**Comment by Compro-Prasad** [comment]
You pushed them in the wrong branch. It is `patch-1` and not `master`. Anyways, I will fix it.

**Pull request merged**